ProteoformSuiteInternal/TdBuReader.cs

```diff
@@ -0,0 +1,222 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+using System.Text.RegularExpressions;
+using UsefulProteomicsDatabases;
+using Proteomics;
+using MzIdentML;
+
+namespace ProteoformSuiteInternal
+{
+    public class TdBuReader
```

Comment:
Could you separate the TD from the BU reader? Also, could you use "TopDown" and "BottomUp" for clarity?